README.md development setup for apple silicon and others (#1454)
vega/editor: README.md

1. Install the dependencies:  
   `$ npm install`

>    * macOS Apple Silicon: if you encounter installation errors related to `canvas`, ensure the necessary system libraries are present by running:   
>      `$ brew install pkg-config cairo pango libpng jpeg giflib librsvg pixman python-setuptools`  
>      For more details, refer to the [canvas installation guide](https://github.com/Automattic/node-canvas?tab=readme-ov-file#installation)

Reviewer: ```suggestion
   * If you are running into issues with installing canvas, follow the [canvas installation guide](https://github.com/Automattic/node-canvas?tab=readme-ov-file#installation).
```
